rllib/algorithms/bc/torch/bc_torch_rl_module.py

```diff
@@ -0,0 +1,60 @@
+from typing import Any, Mapping
+
+from ray.rllib.algorithms.bc.bc_rl_module import BCRLModule
+from ray.rllib.core.models.base import ENCODER_OUT, STATE_OUT
+from ray.rllib.core.rl_module.rl_module import RLModule
+from ray.rllib.core.rl_module.torch.torch_rl_module import TorchRLModule
+from ray.rllib.policy.sample_batch import SampleBatch
+from ray.rllib.utils.annotations import override
+from ray.rllib.utils.framework import try_import_torch
+from ray.rllib.utils.nested_dict import NestedDict
+
+torch, nn = try_import_torch()
+
+
+class BCTorchRLModule(TorchRLModule, BCRLModule):
```

Comment:
The comment for the BCTfRLModule also applies here, please consolidate 🙂

Reply:
Done